### PR TITLE
Changelog for WPCS 2.2.1

### PR DESCRIPTION
**Pull request by jrfnl**
:exclamation: Reviews & approval welcome. I'll merge this myself (if approved) just before the release. :exclamation: 

* Release date set at this ~~**Sunday February 2nd**~~ **Tuesday February 4rd**.
* Includes all currently merged changes + PR #1858.

### CODE REVIEW

**Comment by jrfnl** [comment]
Updated the PR + PR description to include PR #1858 and delayed the target release date to Tuesday Feb 4.

**Comment by jrfnl** [comment]
FYI: Updated for the merge of #1852.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [2.2.1] - 2020-02-04
```
Comment (GaryJones) [inline CHANGELOG.md]: Needs a corresponding link at the bottom of the file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
...
+
+### Added
+- Metrics to the `WordPress.Arrays.CommaAfterArrayItem` sniff. These can be displayed using `--report=info`.
+- The `sanitize_hex_color()` and the `sanitize_hex_color()` functions to the `escapingFunctions` list used by the `WordPress.Security.EscapeOutput` sniff.
```
Comment (GaryJones) [inline CHANGELOG.md]:
```suggestion
- The `sanitize_hex_color()` and the `sanitize_hex_color_no_hash()` functions to the `escapingFunctions` list used by the `WordPress.Security.EscapeOutput` sniff.
```

Comment (jrfnl) [reply CHANGELOG.md]: :fearful: copy/paste fail... Thanks for catching that. Updated now.

File: CHANGELOG.md (hunk 1 above)
Comment (jrfnl) [reply CHANGELOG.md]: :+1: Done

**Pull request merged**